cmd/lgiap/BUILD

```diff
@@ -0,0 +1,18 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library")
+
+go_library(
+    name = "go_default_library",
+    srcs = ["main.go"],
+    importpath = "github.com/livegrep/livegrep/cmd/lgiap",
+    visibility = ["//visibility:private"],
+    deps = [
+        "//server/api:go_default_library",
+        "@com_github_nelhage_go_cli//config:go_default_library",
```

Comment:
nit - you don't use this `go_cli` lib in `lgiap`, can be removed

Reply:
🤯